definitions/pipelines/rnaseq_star.cwl

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "RNA-Seq alignment and transcript/gene abundance workflow"
+requirements:
+    - class: MultipleInputFeatureRequirement
+    - class: SubworkflowFeatureRequirement
+    - class: ScatterFeatureRequirement
+    - class: InlineJavascriptRequirement
+inputs:
+    instrument_data_bams:
+        type: File[]
+    outSAMattrRGline:
+        type:
+            type: array
+            items:
+                type: array
+                items: string
+    genomeDir:
+        type: Directory
+    sjdbGTFfile:
+        type: File
```

Comment:
This isn't a very informative name.  Could you make this a little more human readable?  Is it just the gtf?  If so, maybe `gtf_file` or `transcriptome_gtf_file` or something